blockchain/types/anchoring_data.go

```diff
@@ -60,14 +60,14 @@ func (data *AnchoringDataLegacy) GetBlockNumber() *big.Int {
 }
 
 type AnchoringDataInternalType0 struct {
-	BlockHash     common.Hash
-	TxHash        common.Hash
-	ParentHash    common.Hash
-	ReceiptHash   common.Hash
-	StateRootHash common.Hash
-	BlockNumber   *big.Int
-	BlockCount    *big.Int
-	TxCount       *big.Int
+	BlockHash     common.Hash `json:"blockHash"`
+	TxHash        common.Hash `json:"transactionsRoot"`
+	ParentHash    common.Hash `json:"parentHash"`
+	ReceiptHash   common.Hash `json:"receiptsRoot"`
```

Comment:
```suggestion
	ReceiptHash   common.Hash `json:"receiptHash"`
```

Reply:
I aligned with below `Header`.
```
type Header struct {
	ParentHash  common.Hash    `json:"parentHash"       gencodec:"required"`
	Rewardbase  common.Address `json:"reward"           gencodec:"required"`
	Root        common.Hash    `json:"stateRoot"        gencodec:"required"`
	TxHash      common.Hash    `json:"transactionsRoot" gencodec:"required"`
	ReceiptHash common.Hash    `json:"receiptsRoot"     gencodec:"required"`
	Bloom       Bloom          `json:"logsBloom"        gencodec:"required"`
	BlockScore  *big.Int       `json:"blockScore"       gencodec:"required"`
	Number      *big.Int       `json:"number"           gencodec:"required"`
	GasUsed     uint64         `json:"gasUsed"          gencodec:"required"`
	Time        *big.Int       `json:"timestamp"        gencodec:"required"`
	// TimeFoS represents a fraction of a second since `Time`.
	TimeFoS    uint8  `json:"timestampFoS"     gencodec:"required"`
	Extra      []byte `json:"extraData"        gencodec:"required"`
	Governance []byte `json:"governanceData"        gencodec:"required"`
	Vote       []byte `json:"voteData,omitempty"`
}
```